examples/run-queue.php

```diff
@@ -14,7 +14,7 @@
 $configuration = APNSConfiguration::production( $auth );
 $configuration->setUserAgent( 'wordpress.com development' );
 
-$client = new APNSClient( $configuration );
+$client = new APNSClient( $configuration, new CurlMultiplexedNetworkService() );
```

Comment:
`CurlMultiplexedNetworkService` implements `MultiplexedNetworkService` and uses two DTO types: `Request` and `Response`. 

The names, like the rest of the design, are open for discussion :smile: